horreum-api/src/main/java/io/hyperfoil/tools/horreum/api/services/SchemaService.java

```diff
@@ -239,19 +241,19 @@ int addOrUpdateTransformer(@PathParam("schemaId") int schemaId,
    @Parameters(value = {
            @Parameter(name = "id", description = "Schema ID", example = "101"),
    })
-   @APIResponseSchema(value = String.class,
-           responseDescription = "A JSON representation of the Schema object",
+   @APIResponseSchema(value = SchemaExport.class,
+           responseDescription = "A JSON representation of the SchemaExport object",
            responseCode = "200")
-   String exportSchema(@PathParam("id") int id);
+   SchemaExport exportSchema(@PathParam("id") int id);
 
    @POST
    @Path("import")
    @Consumes(MediaType.APPLICATION_JSON)
-   @RequestBody(content = @Content( mediaType = MediaType.APPLICATION_JSON,
-           schema = @org.eclipse.microprofile.openapi.annotations.media.Schema(
-                   type = SchemaType.STRING, implementation = String.class)) )
-   @Operation(description="Import an previously exported Schema")
-   void importSchema(String config);
+   @APIResponse(responseCode = "204", description = "Import a new Schema or update an existing Schema")
```

Comment:
The response code should be "201" : https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201